src/main/scala/org/broadinstitute/dsde/firecloud/utils/StreamingPassthrough.scala

```diff
@@ -75,6 +75,19 @@ trait StreamingPassthrough
     }
   }
 
+  /**
+   * The Rawls passthrough implementation:
+   *   - Similar to the passthroughImpl except it supplies a remotePath to be used in downstream uri construction
+   */
+  def rawlsPassthrough(localBasePath: Uri.Path, remoteBaseUri: Uri, rawlsPath: String): Route = {
```

Comment:
Rather than having a Rawls-specific passthrough function, I wonder if it would make sense to incorporate this optional remotePath into the existing passthrough. It could be useful in other places.

If we don't want to do that, I suggest renaming this to something like `passthroughWithRemotePathOverride` since it seems generically useful.